src/Akeneo/Pim/Enrichment/Bundle/Command/IndexProductModelCommand.php

```diff
@@ -78,6 +78,12 @@ protected function configure()
                 InputOption::VALUE_NONE,
                 'Index all existing product models into Elasticsearch'
             )
+            ->addOption(
+                'diff',
+                'd',
+                InputOption::VALUE_NONE,
+                'Resolve differences between MySQL and Elasticsearch'
```

Comment:
```suggestion
               ' Index both missing product models present in Mysql and not in ES and outdated product model documents in ES. It does not remove product model documents present in ES but not in Mysql. See pim:product-model:clean-removed-products for that. This option does not work with "all" option. '            
```

Btw, it's missing "pim:product-model:clean-removed-product-models  no ?